src/BladeSDK.ts

```diff
@@ -626,7 +627,7 @@ export class BladeSDK {
      */
     async getKeysFromMnemonic(mnemonicRaw: string, lookupNames: boolean, completionKey?: string): Promise<PrivateKeyData> {
         try {
-            const mnemonic = await Mnemonic.fromString(mnemonicRaw);
+            const mnemonic = await Mnemonic.fromString(mnemonicRaw.toLowerCase());
```

Comment:
 if it's a plain user input, shouldn't we also trim it?

Reply:
```
"         dddd bbb       cccc    ".split(" ").filter(word => word).join(" ")
```

`// 'dddd bbb cccc'`